File: C/1006.c
#include <stdio.h>
int main()
{
    float a,b,c;
    scanf("%f%f%f",&a,&b,&c);
    a *= 2;
    b *= 3;
    c *= 5;
    printf("MEDIA = %.1f\n",(a+b+c)/10);
    return 0;
}

File: PHP/1006.php
<?php
$a = rtrim(fgets(STDIN));
$b = rtrim(fgets(STDIN));
$c = rtrim(fgets(STDIN));
echo "MEDIA = ". number_format((float)((($a*2+$b*3+$c*5)/10)),1,".",'') . "\n";
?>